presto-benchmark/src/main/java/com/facebook/presto/benchmark/SqlConsecutiveJoinBenchmark.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark;
+
+import com.facebook.presto.testing.LocalQueryRunner;
+
+import static com.facebook.presto.benchmark.BenchmarkQueryRunner.createLocalQueryRunner;
+
+public class SqlConsecutiveJoinBenchmark
+        extends AbstractSqlBenchmark
+{
+    public SqlConsecutiveJoinBenchmark(LocalQueryRunner localQueryRunner)
+    {
+        super(localQueryRunner,
+                "sql_consecutive_join",
+                4,
+                5,
+                "SELECT * FROM\n" +
+                        "(\n" +
+                        "  SELECT t1.orderkey AS key, t1.quantity, t2.orderkey, t2.quantity FROM\n" +
+                        "  (SELECT orderkey, quantity FROM lineitem) t1\n" +
+                        "  JOIN\n" +
+                        "  (SELECT orderkey, quantity FROM lineitem) t2\n" +
+                        "  ON t1.orderkey = t2.orderkey\n" +
```

Comment:
That's a good point; I was surprised when I first ran the benchmark and there was no difference in CPU time with or without patch; now I can see a significant improvement.